### PR TITLE
Separate out metrics endpoints to service only per kind

### PR DESCRIPTION
**Pull request by rrati**
part of: https://issues.redhat.com/browse/APPSRE-2910

### CODE REVIEW

File: dashdotdb/schemas/swagger.yaml [hunk 1]
```diff
@@ -7,12 +7,13 @@ info:
   license:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
-  version: 1.0.0
+  version: 1.1.0
 servers:
 - url: /api/v1
 paths:
-  /metrics:
+  /metrics/imagemanifestvuln:
```
Comment (apahim) [inline dashdotdb/schemas/swagger.yaml:14]: Wonder if this shouldn't be the the inverse... we already have a `/imagemanifestvuln` route, I'd just add a `/imagemanifestvuln/metrics` one. Looks more natural... not sure though. No strong opinions. Wdyt?

Comment (apahim) [reply dashdotdb/schemas/swagger.yaml:14]: Same applies to the other ones, os course.

**Review by apahim — APPROVED** [review]
Small comment, looks good either way.

File: dashdotdb/schemas/swagger.yaml (hunk 1 above)
Comment (rrati) [reply dashdotdb/schemas/swagger.yaml:14]: I think I like your suggestion and it is more natural way to extend in the future.  I'll give it a try.

Comment (rporres) [reply dashdotdb/schemas/swagger.yaml]: I agree with @apahim suggestion :)

Comment (rrati) [reply dashdotdb/schemas/swagger.yaml]: Already done.

**Pull request merged**